### ISSUE TITLE
Update Page State Action: Not Clickable

### ISSUE THREAD

**Issue by dinasorz** [FlutterFlow/flutterflow-issues#1888]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Can't update the pageState/pageParameters as it's unclickable, it requires me to add page variables first despite me already added them

### Expected Behavior

To be able to update PageState normally like AppState

### Steps to Reproduce

-Add page parameters
-Choose any button and try to update page state in it, you'll not be able to as it'll require you to add page parameters first

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4mbojPLlvJmbFIocte7q6JccBKjSQmbsHH5+ap0mbnlMpOEC0jmbF7KuBuxzOePRdhNtOMPhaVNcA2CLmbvFge+aHIhmbdrlFPUCA0+BbN1/+ZANbA/0e2imbQRJPXCmbZCrPWhRI/xhG3645mbqun3RElAOhpW1Yt1EymNu6vdADDFOu/Zwh9f6PHaOI=

### Context

Page Parameters are simply unusable like that

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/95580779/312e420b-7ef6-4018-8629-1ac69a27c7ac)
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/95580779/ac5a610b-e2ca-48ff-8823-b1967237269f)

### Environment

```markdown
- FlutterFlow version: 4.0
- Platform: PC
- Browser name and version: Chrome v119.0.6045.160
- Operating system and version affected: Windows 11
```

**Comment by d33p-dev** [comment]
Yeah the docs are broken.  As of Feb 2025, the docs show you a video/steps to attach a a query to a datatable and the query results to a Page State.  But, it doesn't work that way anymore apparently.  I guess you have to add a Page State field first then go through the steps to build an Action / Query / Create List / Name the Variable.  And, I guess at that point you can bind a DataTable to the data....  it's not just confusing for someone new to FF, the docs are just broken....  Nice
